README.md

```diff
@@ -1,41 +1,98 @@
 ## Bridger
 
-[![bridger](https://github.com/darwinia-network/bridger/workflows/bridger/badge.svg)](https://github.com/darwinia-network/bridger)
+[![CI](https://github.com/darwinia-network/bridger/workflows/CI/badge.svg)](https://github.com/darwinia-network/bridger/actions)
 [![crate](https://img.shields.io/crates/v/darwinia-bridger.svg)](https://crates.io/crates/darwinia-bridger)
 [![doc](https://img.shields.io/badge/current-docs-brightgreen.svg)](https://docs.rs/darwinia-bridger/)
 [![downloads](https://img.shields.io/crates/d/darwinia-bridger.svg)](https://crates.io/crates/darwinia-bridger)
 [![LICENSE](https://img.shields.io/crates/l/darwinia-bridger.svg)](https://choosealicense.com/licenses/gpl/)
 
-The Darwinia Bridger
+Darwinia Bridger is the Darwinia relayer client and watchtower written in Rust and Golang.
```

Comment:
Darwinia Bridger is the Darwinia relayer client and watchtower written in Rust.

Reply:
Fixed. @wuminzhe